js/bundle/README.md

```diff
@@ -0,0 +1,29 @@
+# js/bundle
+This directory contains a JavaScript library for serializing (and parsing, in the future) the `application/webbundle` format defined in the [Bundled HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html) draft spec.
```

Comment:
Maybe also write out that currently it doesn't support origin-signed exchanges.

Reply:
Done.